syndeo-template/README.md

```diff
@@ -18,6 +18,50 @@ transforms. Then, the template will take the pipeline spec, and
 
 ## Common workflow tasks
 
+### Format code
+
+To apply spotless formatting rules to the Syndeo template code, run the following command:
+
+```shell
+mvn -B spotless:apply compile -f  unified-templates.xml -pl syndeo-template/pom.xml
+```
+
+### Run tests (unit and integration tests)
+
+To **run unit tests** for the Syndeo template, run the following command. Note that this command knows to skip
+integration tests and only runs unit tests:
+
+```shell
+mvn clean package test -f unified-templates.xml -pl syndeo-template/pom.xml
+```
+
+The file `BigTableWriteIT` **holds integration tests** for the basic BigTable Syndeo integration. These integration tests
+rely on the existence of a BigTable instance and table, as well as a BigQuery dataset, which holds the BQ data that
+is part of the pipeline's read.
+
```

Comment:
May we consider inserting the following?  I don't know if further instructions are needed.  I couldn't complete the instructions to see what else would be needed for setup due to the `Could not resolve dependencies for project com.google.cloud.teleport.syndeo:syndeo-template:jar:1.0-SNAPSHOT` error noted above. 

--------------

To set up your Google Cloud project for the integration tests, the following steps assume you have installed and setup [gcloud](https://cloud.google.com/sdk/gcloud).

1. Set your default project.

```shell
gcloud config set project PROJECT
```

2. Create the BigQuery dataset.

```
bq mk syndeo_dataset
```

3. Create the BigTable instance.

```
gcloud bigtable instances create teleport --display-name=teleport --cluster-config=id=teleport,zone=us-central1-a
```

4. Create the artifact bucket

```
ARTIFACT_BUCKET=[CHANGE ME]
gsutil mb gs://$ARTIFACT_BUCKET
```

5. Create the temporary location bucket

```
TEMP_LOCATION_BUCKET=[CHANGE ME]
gsutil mb gs://$TEMP_LOCATION_BUCKET
```

Reply:
this is awesome. Thanks!